Commonality Analysis Code_2.Rmd

```diff
@@ -0,0 +1,373 @@
+
+boot.regOut_cts_PP <-boot(cope_ca_data_imputed_PP, boot.yhat, 1000, lmOut = cts_PP_imputed, regrout0 = regOut_cts_PP)
+boot.regOut_cts_PP_result <- booteval.yhat(regOut_cts_PP, boot.regOut_cts_PP, bty="perc")
+boot.regOut_cts_PP_result$combCIaps
+
+regOut_cts_PP$APSRelatedMetrics$` % Total`[1:3]
+boot.regOut_cts_PP_result$lowerCIaps$Commonality[1:3] / summary(cts_PP_imputed)$r.squared
+boot.regOut_cts_PP_result$upperCIaps$Commonality[1:3] / summary(cts_PP_imputed)$r.squared
+
+# Project ABC, Depression
+
+regOut_dep_ABC <- calc.yhat(depression_ABC_imputed)
+regOut_dep_ABC$APSRelatedMetrics
+
+boot.regOut_dep_ABC <- boot(cope_ca_data_imputed_ABC, boot.yhat, 1000, lmOut = depression_ABC_imputed, regrout0 = regOut_dep_ABC)
+boot.regOut_dep_ABC_result <- booteval.yhat(regOut_dep_ABC, boot.regOut_dep_ABC, bty = "perc")
+boot.regOut_dep_ABC_result$combCIaps
+
+regOut_dep_ABC$APSRelatedMetrics$` % Total`[1:3]
+boot.regOut_dep_ABC_result$lowerCIaps$Commonality[1:3] / summary(depression_ABC_imputed)$r.squared
+boot.regOut_dep_ABC_result$upperCIaps$Commonality[1:3] / summary(depression_ABC_imputed)$r.squared
+
+# Project ABC, Anxiety
+
+regOut_anx_ABC <- calc.yhat(anxiety_ABC_imputed)
+regOut_anx_ABC$APSRelatedMetrics
+
+boot.regOut_anx_ABC <- boot(cope_ca_data_imputed_ABC, boot.yhat, 1000, lmOut = anxiety_ABC_imputed, regrout0 = regOut_anx_ABC)
+boot.regOut_anx_ABC_result <- booteval.yhat(regOut_anx_ABC, boot.regOut_anx_ABC, bty = "perc")
+boot.regOut_anx_ABC_result$combCIaps
+
+regOut_anx_ABC$APSRelatedMetrics$` % Total`[1:3]
+boot.regOut_anx_ABC_result$lowerCIaps$Commonality[1:3] / summary(anxiety_ABC_imputed)$r.squared
+boot.regOut_anx_ABC_result$upperCIaps$Commonality[1:3] / summary(anxiety_ABC_imputed)$r.squared
+
+# Project ABC, Restrictive Eating
+
+regOut_drs_ABC <- calc.yhat(drs_ABC_imputed)
+regOut_drs_ABC$APSRelatedMetrics
+
+boot.regOut_drs_ABC <- boot(cope_ca_data_imputed_ABC, boot.yhat, 1000, lmOut = drs_ABC_imputed, regrout0 = regOut_drs_ABC)
+boot.regOut_drs_ABC_result <- booteval.yhat(regOut_drs_ABC, boot.regOut_drs_ABC, bty = "perc")
+boot.regOut_drs_ABC_result$combCIaps
+
+regOut_drs_ABC$APSRelatedMetrics$` % Total`[1:3]
+boot.regOut_drs_ABC_result$lowerCIaps$Commonality[1:3] / summary(drs_ABC_imputed)$r.squared
+boot.regOut_drs_ABC_result$upperCIaps$Commonality[1:3] / summary(drs_ABC_imputed)$r.squared
+
+# Project ABC, COVID-19 Trauma
+
+regOut_cts_ABC <- calc.yhat(cts_ABC_imputed)
+regOut_cts_ABC$APSRelatedMetrics
+
+boot.regOut_cts_ABC <- boot(cope_ca_data_imputed_ABC, boot.yhat, 1000, lmOut = cts_ABC_imputed, regrout0 = regOut_cts_ABC)
+boot.regOut_cts_ABC_result <- booteval.yhat(regOut_cts_ABC, boot.regOut_cts_ABC, bty = "perc")
+boot.regOut_cts_ABC_result$combCIaps
+
+regOut_cts_ABC$APSRelatedMetrics$` % Total`[1:3]
+boot.regOut_cts_ABC_result$lowerCIaps$Commonality[1:3] / summary(cts_ABC_imputed)$r.squared
+boot.regOut_cts_ABC_result$upperCIaps$Commonality[1:3] / summary(cts_ABC_imputed)$r.squared
+
+```
+
+# Visualizing
+

```

Comment:
Super plot! Everything looks good here at first glance, but let's take a closer look once the other things are addressed